### DISCUSSION TITLE
Is there any possible to get the generated class by Master CSS ?

### DISCUSSION THREAD

**Discussion by jackyyangtw** [master-co/css#59] [Help]
I have an iframe page which built width Master css cdn, but the parent site rendering the iframe content strangely, it looks like some conflict with parent page and Master css. Is there any possible to get the generated class by Master CSS(maybe to generate a css file)? Or is there any other solution for this case ?

**Comment by 1aron** [comment]
@jackyyangtw Are you referring to the source of the styles used in the iframe come from CSS rules generated by the parent window rather than the rules generated according to the class name in the iframe?

**Reply by jackyyangtw** [reply]
My iframe page is not using CSS rules generated by the parent window, just using the Master CSS CDN to style it. And my parent page is not load the Master CSS CDN, just the iframe page load that.

**Comment by 1aron** [comment]
@jackyyangtw Master CSS JIT mode observes changes in DOM class names on the browser side to instantly generate corresponding CSS rules and inject them into running style sheets.

**Comment by 1aron** [comment]
We're also currently working on an AOT mode that allows you to generate a static CSS file.

**Reply by jackyyangtw** [reply]
So I can get the css file generated by Master CSS in future? That will be helpful!
